From c2d75cdcd7e2c426d5813daec8a91cbd05d5811d Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Tue, 29 Aug 2017 22:58:16 +0100
Subject: [PATCH] wm: Temporarily disable WM maximize animation per issue #1129

This is just to provide a patch for the issue, and we'll find a fix
in the mean time.

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 src/wm/wm.vala | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/wm/wm.vala b/src/wm/wm.vala
index 83083aea..708e3070 100644
--- a/src/wm/wm.vala
+++ b/src/wm/wm.vala
@@ -711,6 +711,7 @@ public class BudgieWM : Meta.Plugin
         actor.restore_easing_state();
     }
 
+    /*
     public override void size_change(Meta.WindowActor actor, Meta.SizeChange which_change, Meta.Rectangle old_frame_rect, Meta.Rectangle old_buffer_rect)
     {
         if (!this.use_animations) {
@@ -863,7 +864,7 @@ public class BudgieWM : Meta.Plugin
         snapshot.content = canvas;
 
         return snapshot;
-    }
+    }*/
 
     void minimize_done(Clutter.Actor? actor)
     {
